packages/sdk/tests/e2e/sharing/removeProtectedDataFromRenting.test.ts

```diff
@@ -43,62 +42,36 @@ describe('dataProtector.removeProtectedDataFromRenting()', () => {
 
         await dataProtector.setProtectedDataToRenting({
           protectedDataAddress: result.address,
-          collectionTokenId: collectionTokenId,
           durationInSeconds: duration,
           priceInNRLC: price,
         });
         await sleep(2000);
         const { success } = await dataProtector.removeProtectedDataFromRenting({
-          collectionTokenId: collectionTokenId,
           protectedDataAddress: result.address,
         });
         expect(success).toBe(true);
       },
       10 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME
     );
-    it(
-      'should fail with collection does not exist error',
-      async () => {
-        //create a random protected data address
-        const protectedDataAddressMock = Wallet.createRandom().address;
-        //generate a random collection id that dosn't exist
-        const min = 1000000;
-        const max = Number.MAX_SAFE_INTEGER;
-        const randomCollectionTokenId =
-          Math.floor(Math.random() * (max - min + 1)) + min;
-        await expect(() =>
-          dataProtector.removeProtectedDataFromRenting({
-            collectionTokenId: randomCollectionTokenId,
-            protectedDataAddress: protectedDataAddressMock,
-          })
-        ).rejects.toThrow(
-          new WorkflowError(
-            'Failed to Remove Protected Data From Renting: collection does not exist.'
-          )
-        );
-      },
-      2 * MAX_EXPECTED_BLOCKTIME + MAX_EXPECTED_WEB2_SERVICES_TIME
-    );
     it(
       'should fail with not collection owner error',
```

Comment:
Expect changed but not the description, should it rather be:

```suggestion
      'should fail with protected data does not exist',
```